### PR TITLE
Added LambdaTest in the list of  cloud emulators

### PR DESCRIPTION
**Pull request by jahmed2345**

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers.md [hunk 1]
```diff
@@ -120,6 +120,7 @@ Microsoft Edge \(EdgeHTML\) Virtual Machines \(VMs\) enable you to access differ
 If you are not able to use the emulators and do not have access to real devices, then cloud-based emulators are the next best thing.  A big advantage of cloud-based emulators over real devices and local emulators is that you are able to automate unit tests for your site across different platforms.  
 
 *   [BrowserStack (commercial)][BrowserStack] is the easiest to use for manual testing.  You select an operating system, select your browser version and device type, select a URL to browse, and it spins up a hosted virtual machine with which you may interact.  You are able to also run multiple emulators in the same screen, enabling you to test the look and feel of your app across multiple devices at the same time.  
+*   [LambdaTest (commercial)][LambdaTest] will help you to perform manual cross browser testing on a combination of 2000+ browsers & operating systems. Users will be able to record video of complex bugs and even share it via integrations like MS Teams, Slack and more. Users can speed-up their testing by running tests in parallel.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers.md:123]:
```suggestion
*   [LambdaTest (commercial)][LambdaTest] helps you perform manual cross-browser testing on a combination of 2000+ browsers and operating systems. Users can record videos of complex bugs and even share them through integration such as Microsoft Teams, Slack and more. Users can speed-up their testing by running tests in parallel.
```
An Edit pass.

File: microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers.md [hunk 2]
```diff
@@ -120,6 +120,7 @@ Microsoft Edge \(EdgeHTML\) Virtual Machines \(VMs\) enable you to access differ
 If you are not able to use the emulators and do not have access to real devices, then cloud-based emulators are the next best thing.  A big advantage of cloud-based emulators over real devices and local emulators is that you are able to automate unit tests for your site across different platforms.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers.md:120]: probably should update the date at top of file.

**Review by mikehoffms — APPROVED** [review]

**Comment by jahmed2345** [comment]
@mikehoffms when will this request be merged?

**Comment by mikehoffms** [comment]
> @mikehoffms when will this request be merged?

On Friday, I asked Zoher to approve this PR.  ETA merge early this week.

**Review by zoherghadyali — APPROVED** [review]

**Pull request merged**